devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/language/kotlin/codestart.yml

```
@@ -5,4 +5,6 @@ language:
     shared-data:
       language:
         dir:
-          code: src/main/kotlin
\ No newline at end of file
+          code: src/main/kotlin
+    dependencies:
+      io.quarkus:quarkus-scala
```

It should be `quarkus-kotlin` here no ?

---

```suggestion
      io.quarkus:quarkus-kotlin
```

@glefloch very good catch indeed!!!